shared/templates/selinux_booleans.csv

```diff
@@ -2,15 +2,15 @@
 # Add <selinux_boolean_name, use_var> to generate OVAL content that use XCCDF values.
 abrt_anon_write,
 abrt_handle_event,
-abrt_upload_watch_anon_write,
-antivirus_can_scan_system,
+abrt_upload_watch_anon_write,disable
```

Comment:
@redhatrises 
Can you clarify where the request to 'disable' this boolean comes from? (is there some official guidance recommending this to be disabled?)

Asking, because default RHEL-7 setting has it enabled:
```
# semanage boolean -l | grep abrt_upload_watch_anon_write
abrt_upload_watch_anon_write   (on   ,   on)  Allow abrt to upload watch anon write
```